mkdocs.yml

```diff
@@ -229,7 +229,7 @@ extra_javascript:
 
 markdown_extensions:
   - toc:
-      toc_depth: 2-6
+      toc_depth: 1-6
```

Comment:
> What was the reason for this change?

This is part of another PR, just shows up here because I branched off of it here by mistake. I might just redo this PR after rebasing or something like that.